mindsdb/integrations/handlers/tripadvisor_handler/json_response_examples/location_details.json

```diff
@@ -0,0 +1,197 @@
+{
```

Comment:
Hey @HahaBill,
I think we discussed that these JSON files can be removed.

Reply:
Oh I thought I've already removed it. Will do.